src/oscillator.js

```diff
@@ -497,8 +497,9 @@ define(function (require) {
    *  its method <code>setType('sine')</code>.
    *  See p5.Oscillator for methods.
    *
-   *  @method  p5.SinOsc
-   *  @param {[Number]} freq Set the frequency
+   *  @method  SinOsc
```

Comment:
these are actually constructors for classes that inherit from p5.Oscillator, and using `@ method` was just a hack to get them to appear on the p5.Oscillator page.

Do you happen to know a good way to ensure that these appear on the p5.Oscillator page, but not the p5 sound reference?